From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pavel Safar <safar.pavel@gmail.com>
Date: Mon, 7 Jun 2021 14:26:26 +0200
Subject: [PATCH] Skip using find_package() for SDL2

We pass the include directory via cmake flags, so we can just use that
and link against sdl2 normally.
---
 CMakeLists.txt | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 38f6ba8..d0d8463 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,8 +24,7 @@ add_executable(raw
         src/vm.cpp
 )
 
-find_package(SDL2 REQUIRED)
-include_directories(${SDL2_INCLUDE_DIRS})
-target_link_libraries(raw ${SDL2_LIBRARIES})
+include_directories(${SDL2_INCLUDE_DIR})
+target_link_libraries(raw SDL2)
 target_link_libraries(raw z)
 
